src/main/webapp/app/entities/flair-bi/filter/filter-element-grpc.component.js

```diff
@@ -264,13 +266,26 @@
             query.distinct = true;
             query.limit = 100;
             favouriteFilterService.setFavouriteFilter(isFavouriteFilter());
-            proxyGrpcService.forwardCall(
-                vm.view.viewDashboard.dashboardDatasource.id, {
-                queryDTO: query,
-                vId: vId
-            },
-                $stateParams.id
-            );
+            if(vm.sharefilter){
+                proxyGrpcService.forwardCall(
+                    vm.view.viewDashboard.dashboardDatasource.id, {
```

Comment:
Can make this call from share link controller itself.. We do not need make changes in all these files